src/test/java/com/saralein/server/request/RequestParserTest.java

```diff
@@ -3,10 +3,14 @@
 import java.util.HashMap;
 import static org.junit.Assert.*;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
 public class RequestParserTest {
     private RequestParser requestParser;
+    @Rule
+    public final ExpectedException exception = ExpectedException.none();
```

Comment:
What is this rule thing?

Reply:
That was an experiment I tried and forgot to remove.  Will remove it.